mp/src/game/server/neo/neo_player.cpp

```diff
@@ -1096,14 +1097,14 @@ void CNEO_Player::PostThink(void)
 			{
 				static_cast<CWeaponGrenade*>(pNeoWep)->SecondaryAttack();
 			}
-			else if (ClientWantsAimHold(this))
+			else
```

Comment:
There's a mismatch between the client and server here. We're not checking if sprint is being held, which means it's possible to start aiming while sprinting.

I'd probably take the time to fix #276 and #355 in the same PR.

Reply:
Added sprint fixes in PR, although I'm not going to change shared code stuff for the time being as that's more of a wider look at c_neo_player/neo_player split than just this.